pkg/compactor/compactor.go

```diff
@@ -255,6 +259,9 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.DurationVar(&cfg.BlockVisitMarkerTimeout, "compactor.block-visit-marker-timeout", 5*time.Minute, "How long block visit marker file should be considered as expired and able to be picked up by compactor again.")
 	f.DurationVar(&cfg.BlockVisitMarkerFileUpdateInterval, "compactor.block-visit-marker-file-update-interval", 1*time.Minute, "How frequently block visit marker file should be updated duration compaction.")
 
+	f.DurationVar(&cfg.CleanerVisitMarkerTimeout, "compactor.cleaner-visit-marker-timeout", 10*time.Minute, "How long cleaner visit marker file should be considered as expired and able to be picked up by cleaner again.")
```

Comment:
In which case, we need to configure this flag? Can we avoid the flag as if this is a sane default?

Reply:
If cleaner interval is set to be smaller than 10 minutes, this config needs to be adjusted. Otherwise, there could be skipping one cleaner cycle because visit marker is not expired.

Reply:
I see I understand this now. Can we update the flag description to mention this?